sql_request_abstract/models/sql_request_mixin.py

```diff
@@ -296,7 +296,7 @@ def _check_execution(self):
         is done after."""
         self.ensure_one()
         query = self._prepare_request_check_execution()
-        rollback_name = self._create_savepoint()
+        rollback_name = self._create_savepoint(self.env.cr)
```

Comment:
Thanks for the fix
To be consistent with last introduced feature, it think it should be : 
```
query_cr = self._get_cr_for_query()
rollback_name = self._create_savepoint(query_cr)
```

Reply:
Pushed it that way, please check.